database/fingertips-db/Tables/AgeDimension.sql

```diff
@@ -0,0 +1,9 @@
+﻿CREATE TABLE [dbo].[AgeDimension](
+	[AgeKey] [smallint] IDENTITY(1,1) NOT NULL,
+	[Name] [nvarchar](50) NOT NULL,
+	[AgeID] [smallint] NOT NULL,
```

Comment:
Is the intent that the `AgeID` maps to the ID value used in the current Fingertips? I'm assuming that's the case with the `IndicatorId` and `SexId` fields too? Are these scripts the place to document that (i.e. with a comment)?

Reply:
added comment for all tables